css/style.css

```diff
@@ -0,0 +1,615 @@
+*,
+*::after,
+*::before {
+  box-sizing:border-box }
+
+body {
+  max-width: 1350px;
+  margin: 0 auto!important;
+  background-color: rgb(253, 249, 249)!important;
+}
+
+html, body {
+    overflow-x: hidden;
+}
+.container-fluid {
+    position: relative;
+    background-color: white
+}
+
+.heading-none{
+    display: none
+}
+
+.nav-bar {
+	position: relative;
+	margin: auto;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.